internal/bundle/create_repo.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+mkdir -p ../internal/bundle/testrepo ; cd ../internal/bundle/testrepo
```

Comment:
This should all be done within the confines of a single tmp dir.

```
mktemp -d
```

Then change directory to there using pushd, all operations relative to that, then popd, and delete the mktemp -d.

You can get the name of the temp dir that mktemp creates with

```
TEMPDIR=$(mktemp -d)
```

Reply:
Or, create the temp dir in the Ginkgo test, and pass it in to the script. Then you can clean it up in the ginkgo test without the need for the delete_repo.sh script.